src/screens/AccountScreens/ProfileScreen/index.js

```diff
@@ -1,11 +1,56 @@
 import React from 'react';
 import { styles } from './style';
-import { View, Text } from 'react-native';
+import { View, Text, Image, } from 'react-native';
+import { Button, Input, Loading } from '../../../components';
+import { useAuthStore } from '../../../shared/zustand/auth';
+import { THEME_COLORS } from '../../../components/theme';
+
+
 
 const Profile = () => {
+  const { isLoading } = useAuthStore();
+  if (isLoading) {
+    return <Loading />;
+  }
   return (
     <View style={styles.container}>
-      <Text style={{fontSize: 30,}}>Profile Page</Text>
+      <Image
+        source={{
+          uri: "https://w.wallhaven.cc/full/k7/wallhaven-k7xwjm.jpg"
+        }}
+        style={styles.image}
+      ></Image>
+      <View style={styles.userInfoContainer}>
+        <View  style={styles.userInfoTextContainer}>
+        <Text style={styles.userInfoText}>
+          User's Email
+          </Text>
+        </View>
+        <View style={styles.userInfoTextContainer}>
+        <Text style={styles.userInfoText}>
+          Username
+          </Text>
+        </View>
+      </View>
+
+      <View style={styles.buttonGroup}>
+      <Button
+            title="Update my profile"
```

Comment:
Actually, this can create some confusion, as the name & email are not changed yet, how could you update? My suggestion is that we call it `Edit my profile`. When the user clicks the edit button, the text fields become editable, and the title of the button can then change to `Update my profile`